Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/preprocessor/LibraryInfo.kt

```diff
@@ -0,0 +1,261 @@
+package org.cqframework.cql.cql2elm.preprocessor
+
+import org.antlr.v4.runtime.misc.Interval
+import org.antlr.v4.runtime.tree.ParseTree
+import org.cqframework.cql.cql2elm.ResultWithPossibleError
+import org.cqframework.cql.gen.cqlParser.ContextDefinitionContext
+import org.cqframework.cql.gen.cqlParser.LibraryDefinitionContext
+import org.hl7.elm.r1.OperandDef
+
+@Suppress("TooManyFunctions")
+class LibraryInfo : BaseInfo() {
+    var namespaceName: String? = null
+    @JvmField var libraryName: String? = null
+    var version: String? = null
+    var defaultUsingDefinition: UsingDefinitionInfo? = null
+        private set
+
+    private val usingDefinitions: MutableMap<String?, UsingDefinitionInfo>
```

Comment:
issue: incorrect inference for these maps

Reply:
Changed the initializers to
```private val usingDefinitions = LinkedHashMap<String, UsingDefinitionInfo>()
private val includeDefinitions = LinkedHashMap<String, IncludeDefinitionInfo>()
private val codesystemDefinitions = LinkedHashMap<String, CodesystemDefinitionInfo>()
private val valuesetDefinitions = LinkedHashMap<String, ValuesetDefinitionInfo>()
private val codeDefinitions = LinkedHashMap<String, CodeDefinitionInfo>()
private val conceptDefinitions = LinkedHashMap<String, ConceptDefinitionInfo>()
private val parameterDefinitions = LinkedHashMap<String, ParameterDefinitionInfo>()
private val expressionDefinitions = LinkedHashMap<String, ExpressionDefinitionInfo>()
private val functionDefinitions = LinkedHashMap<String, MutableList<FunctionDefinitionInfo>>()
private val contextDefinitions = ArrayList<ContextDefinitionInfo>()
private val definitions = HashMap<Interval, BaseInfo>()
```